YR/Sailpoint-IdentityNow-Collector/CIAC-6503 (#34070)
demisto/content: Packs/SailPointIdentityNow/Integrations/SailPointIdentityNowEventCollector/SailPointIdentityNowEventCollector_description.md

## SailPoint IdentityNow Event Collector

**Identity Now Server URL**:

> follow this link [link](https://developer.sailpoint.com/docs/api/getting-started/#find-your-tenant-name) to find out the tenant name.

Reviewer: ```suggestion
In order to get tenant name, follow this [link](https://developer.sailpoint.com/docs/api/getting-started/#find-your-tenant-name)
```

**Client ID and Client Secret**:

> follow this [link](https://developer.sailpoint.com/docs/api/authentication/#generate-a-personal-access-token) for more information of how to generate client id and client secret.
> 

Reviewer: ```suggestion
In order to generate client id and client secret, follow this [link](https://developer.sailpoint.com/docs/api/authentication/#generate-a-personal-access-token).

```

## SailPoint IdentityNow Event Collector

**Identity Now Server URL**:

> In order to get tenant name, follow this [link](https://developer.sailpoint.com/docs/api/getting-started/#find-your-tenant-name).

Reviewer 2: ```suggestion
In order to get the tenant name, follow this [link](https://developer.sailpoint.com/docs/api/getting-started/#find-your-tenant-name).
```

**Client ID and Client Secret**:

> In order to generate Client ID and Client Secret, follow this [link](https://developer.sailpoint.com/docs/api/authentication/#generate-a-personal-access-token).  

Reviewer 2: ```suggestion
In order to generate the Client ID and Client Secret, follow this [link](https://developer.sailpoint.com/docs/api/authentication/#generate-a-personal-access-token).  
```
